tests/functional/test_all.py

```diff
@@ -252,7 +252,14 @@ def test_execution_plan(self):
 
         result = redis_graph.execution_plan("MATCH (r:Rider)-[:rides]->(t:Team) WHERE t.name = $name RETURN r.name, t.name, $params", {'name': 'Yehuda'})
         expected = "Results\n    Project\n        Conditional Traverse | (t:Team)->(r:Rider)\n            Filter\n                Node By Label Scan | (t:Team)"
-        self.assertEqual(result, expected)
+        self.assertEqual(str(result), expected)
+
+        expected = {'children': [{'children': [{'children': [{'children': [{'op': 'Node By Label Scan | (t:Team)'}],
+                                                              'op': 'Filter'}],
+                                                'op': 'Conditional Traverse | (t:Team)->(r:Rider)'}],
+                                  'op': 'Project'}],
+                    'op': 'Results'}
```

Comment:
We should provide a few utility functions to construct a `ExecutionPlan` instance. this construction revels the internals of the class.